NEWS

```diff
@@ -32,9 +32,22 @@ This file contains notable changes between releases for each release since
 Open MPI 1.0.  Because Open MPI maintains multiple release branches at any time,
 it is possible for items to appear more than once in the list.
 
-3.0.1 -- October, 2017
+3.0.1 -- December, 2017
 ----------------------
 
+- Add UCX to list of interfaces that OpenSHMEM will use by default
+
+- Add --{enable|disable}-show-load-errors-by-default to control
+  default behavior of the load errors option
+
+- OFI MTL improvements: handle empty completion queues properly, fix
+  incorrect error message around fi_getinfo(), use default progress
+  option for provider by default.
+
+- PSM2 MTL improvements: Allow use of GPU buffers, thread fixes
```

Comment:
Nit: some of the lines have periods, so do not.  This is super nit-picky, so I'll just push an update that fixes it. 😄

Reply:
Fixed.  In other versions, we also didn't have blank lines between items, so fixed that.

Reply:
Thanks!  I was trying to use `hub` to check out this issue on my newly-reinstalled Mac and was struggling with getting MacPorts to install `hub` to check out this PR's branch (https://trac.macports.org/ticket/40518).  Sigh.